odoo/models.py

```diff
@@ -831,10 +837,9 @@ def _build_model_attributes(cls, pool):
             for mname, fnames in base._depends.items():
                 depends.setdefault(mname, []).extend(fnames)
 
-            for cons in base._sql_constraints:
-                cls._sql_constraints[cons[0]] = cons
-
-        cls._sql_constraints = list(cls._sql_constraints.values())
+            sql_objects.update(base._sql_objects or base._local_sql_objects)
+        sql_objects.update(cls._local_sql_objects)
+        # XXX cls._sql_objects = sql_objects
```

Comment:
I don't understand this comment. Where do you assign `sql_objects` then? 

Deadcode since you do it also in _setup_base ?

Reply:
Deadcode, leftover from a first attempt.